src/components/EvaluateAndSubmit/EvaluateAndSubmit.js

```
@@ -527,7 +527,7 @@ export const EvaluateAndSubmit = ({
         modalType.current = "error";
         modalHeading.current = "Error";
         modalMessage.current =
-        e.response?.data?.message ?? "Error: An error occurred during the file queueing process. Please try again. If you continue to encounter an error, please contact ECMPS Support.";
+          "Error: An error occurred during the file queueing process. Please try again. If you continue to encounter an error, please contact ECMPS Support.";
         setShowSuccessModal(true);
```

It looks like this change is no longer looking at e.response?.data?.message.  Is this intentional?

---

Yes, this change is intentional. The update removes the dependency on exposureData?.data?.message to avoid potential issues where the message may not always be populated as expected. Instead, the new logic ensures that submission errors are properly evaluated without relying on that specific data structure. And if there’s a use case where this reference is still required, I can make the adjustment.

---

@spetros-do I think we should leave this line as is. Your response did not address why we are removing the dependency on exposureData?.data?.message.

---

@yonatan-dp I am not sure I understand what you mean, "Your response did not address why we are removing the dependency on exposureData?.data?.message." I am providing you my specific explanation why I suggest the changes made. 
Again, the purpose of removing exposureData?.data?.message is to prevent reliance on a potentially inconsistent data structure. For example, in some cases, message may not always be present, which could lead to missing error handling. 

The new logic ensures that the error message is always properly set, regardless of whether message exists in exposureData?.data?.message. If this specific reference is still required for certain scenarios, advise and I can revert it accordingly.

---

@spetros-do 
So, the old logic sets the modal message to `e.response?.data?.message`.  If there is no data in the exception message, then it displays the standard message of ""Error: An error occurred during the file queueing process. Please try again. If you continue to encounter an error, please contact ECMPS Support."

The new change, by removing `e.response?.data?.message` will always display the standard message, regardless of what the exception message is.  Given that this specific change is not related to the ticket description, I am asking if this is intentional (to hide the exception message). I am only being careful that we won't brake something that was put in intentionally.